### ISSUE TITLE
libssl.so.1.1: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by Zeex** [timvisee/ffsend#119]
When running ffsend from a Travis CI build (Ubuntu 16.04.6 LTS), it can't find the OpenSSL library for some reason, even though I've installed the packages via:

```
apt-get install openssl ca-certificates
```

Here is an example build:

https://travis-ci.org/github/Zeex/samp-plugin-crashdetect/builds/705098889#L1495 (scroll towards the very end to see the error)

**Comment by timvisee** [comment]
Please install the following packages instead:

```bash
apt install build-essential cmake pkg-config libssl-dev ca-certificates
```

Some might be pre-installed on Travis CI already, but apt-installing them again won't hurt.

See the build section in the README for more information: https://github.com/timvisee/ffsend#build

**Comment by Zeex** [comment]
Unfortunately that didn't hlep
https://api.travis-ci.org/v3/job/705101435/log.txt

I chose to install ffsend via snap instead as it's easier. Works flawlessly.

**Comment by timvisee** [comment]
Whoops, just noticed your issue wasn't about building, but just about running ffsend.

I think the OpenSSL library available on Ubuntu 16.04 is outdated. Using a newer Ubuntu version for building might be your best option if possible. Otherwise installing a newer OpenSSL version would do the trick, possibly through a PPA.

> I chose to install ffsend via snap instead as it's easier. Works flawlessly.

Good to hear that works fine!
